backend/src/test/java/gov/cdc/usds/simplereport/api/devicetype/DeviceTypeDataLoaderHelperTest.java

```diff
@@ -0,0 +1,115 @@
+package gov.cdc.usds.simplereport.api.devicetype;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import gov.cdc.usds.simplereport.db.model.DeviceSupportedDisease;
+import gov.cdc.usds.simplereport.db.model.DeviceTypeSpecimenTypeMapping;
+import gov.cdc.usds.simplereport.db.model.SpecimenType;
+import gov.cdc.usds.simplereport.db.model.SupportedDisease;
+import gov.cdc.usds.simplereport.db.repository.DeviceSpecimenTypeNewRepository;
+import gov.cdc.usds.simplereport.db.repository.DeviceSupportedDiseaseRepository;
+import gov.cdc.usds.simplereport.db.repository.SpecimenTypeRepository;
+import gov.cdc.usds.simplereport.service.DiseaseService;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+class DeviceTypeDataLoaderHelperTest {
+
+  @Mock DeviceSupportedDiseaseRepository deviceSupportedDiseaseRepository;
+  @Mock DiseaseService diseaseService;
+  @Mock DeviceSpecimenTypeNewRepository deviceSpecimenTypeNewRepository;
+  @Mock SpecimenTypeRepository specimenTypeRepository;
+
+  @InjectMocks private DeviceTypeDataLoaderService deviceTypeDataLoaderService;
+
+  @Test
+  void getSupportedDiseases_happyPath() {
+    // GIVEN
+    UUID device1Id = UUID.randomUUID();
+    UUID device2Id = UUID.randomUUID();
+    UUID device3Id = UUID.randomUUID();
+    Set<UUID> deviceSet = Set.of(device1Id, device2Id, device3Id);
+    UUID supportedDisease1Id = UUID.randomUUID();
+    UUID supportedDisease2Id = UUID.randomUUID();
+
+    SupportedDisease supportedDisease2 = mock(SupportedDisease.class);
+    SupportedDisease supportedDisease1 = mock(SupportedDisease.class);
```

Comment:
nit: I don't know why this bugs me so much but can you put these in order 😆

Reply:
Fixed :D